docs/data/README.mdx

```diff
@@ -10,7 +10,7 @@ This section will walk you through the differences between the various platforms
 
 - **[RPC](#rpc)** - live network gateway
 - **[Horizon](#horizon)** - API for network state data
-- **Galexie** - exports raw ledger metadata files
+- **[Galexie](#Galexie)** - exports raw ledger metadata files
```

Comment:
The link isn't working in the preview for me. I think the `#Galexie` needs to be lowercase `#galexie`.